push-api/JSONParser/README.md

```diff
@@ -1,14 +1,28 @@
-### General usage:
+# Crawl and Push
+
+Scanning a folder and push every item to the Push-API
+
+### Usage
 
 The main file is JSONParser.py
+It will execute a 4 step process:
 
-You will need to add the PyYAML lib
+1: read the config file (yamlParser.py)
```

Comment:
This is markdown, see the preview, it renders horribly ;)

Ordered list are done with 
```
1.
2.
3.
```
(with a dot)